pulsar-broker/src/main/java/com/yahoo/pulsar/broker/namespace/OwnershipCache.java

```diff
@@ -130,9 +127,16 @@
                             if (LOG.isDebugEnabled()) {
                                 LOG.debug("Successfully acquired zk lock on {}", namespaceBundleZNode);
                             }
+                            final OwnedBundle bundle = new OwnedBundle(
+                                    ServiceUnitZkUtils.suBundleFromPath(namespaceBundleZNode, bundleFactory));
+                            // update the version
+                            localZkCache.getZooKeeper().exists(path, false, (rc1, path1, ctx1, stat) -> {
```

Comment:
> now pervious broker-1 restarts and it still has bundle in cache so, while unloading bundle it deletes the znode

What do you mean by this? If broker-1 restarts, the z-node will go away and the in-memory cache is gone.